### PR TITLE
Update Game Patch Zip URL

### PR DESCRIPTION
**Pull request by illusionyy**
New repo lives here: https://github.com/illusion0001/PS4-PS5-Game-Patch
Tested on my PS4, it downloads and extracts work.

### CODE REVIEW

**Review by bucanero — APPROVED** [review]

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -42,58 +42,58 @@ jobs:
...
         repository: GoldHEN/GoldHEN_Cheat_Repository
         path: cheats
 
     - name: Checkout Patch repository
-      uses: actions/checkout@v3
+      uses: actions/checkout@main
       with:
         repository: GoldHEN/GoldHEN_Patch_Repository
         path: appdata1
         ref: gh-pages
```
Comment (bucanero) [inline .github/workflows/build.yml]: Since we are moving to your new repo, 
Should this step be changed too?

**Review by bucanero — COMMENTED** [review]
I think the workflow needs a repo update too, so the .pkg gets built with the new patch files.

btw, I'll have to do a quick release once we merge this because the URL is hardcoded in the binary 😅

File: .github/workflows/build.yml (hunk 1 above)
Comment (illusionyy) [reply .github/workflows/build.yml]: Yes, I will download it using curl in the appdata step.

Comment (illusionyy) [reply .github/workflows/build.yml]: Simplified it more by unzipping via BusyBox for stdin support

**Review by bucanero — APPROVED** [review]

**Pull request merged**